app/assets/stylesheets/components/footer.scss

```diff
@@ -1,190 +1,154 @@
-@include only-print {
-  body {
-    position: relative;
-  }
-
-  .footer {
-    page-break-after: always;
-  }
-}
-
 @include b(footer) {
   background-color: $navy-blue;
   color: $white;
 
-  @include only-print {
-    position: static;
-    bottom: 0;
+  @include e(body) {
+    padding-top: vr(2);
+    padding-bottom: vr(2);
   }
 
+  @include e(footer) {
+    background-color: $teal;
+  }
 
-  @include e(content) {
-    overflow: hidden;
-
+  @include e(inner) {
+    margin: 0 auto;
     width: $main__width;
-    margin: auto;
-    padding-top: vr(2);
 
     @include mobile-container;
-
-    @include only-print {
-      width: 80%;
-    }
   }
 
-  @include e(navigation) {
-    background-color: $prussian-blue;
-    line-height: vr(2);
+  @include e(items) {
+    @media (min-width: $break__desktop) {
```

Comment:
Is this the same as only_desktop()?

Reply:
No, this is a new breakpoint I had to include for the header and footer.  (`$break__desktop: 1280px;`)

The mixin only_desktop works like this:

`@mixin only-desktop {
  @media (min-width: $break__phone) {
    @content;
  }
}`

Reply:
Oh I see, sorry should have checked!